index.html

```diff
@@ -0,0 +1,432 @@
+                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed malesuada vulputate
+                                    nisi in
+                                    fermentum. Vivamus ac libero quis nisi auctor pulvinar. Aenean sit amet lectus
+                                    posuere,
+                                    mattis massa eget, ullamcorper diam. Nunc sit amet felis eget arcu congue dictum.
+                                </p>
+                                <q class="title">John Doe</q>
+                            </div>
+                            <div class="testimonial">
+                                <div class="picure">
+                                    <img src="resources/testimonial-img2.jpg" alt="Jenny Martian Photo">
+                                </div>
+                                <p class="description">
+                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed malesuada vulputate
+                                    nisi in
+                                    fermentum. Vivamus ac libero quis nisi auctor pulvinar. Aenean sit amet lectus
+                                    posuere,
+                                    mattis massa eget, ullamcorper diam. Nunc sit amet felis eget arcu congue dictum.
+                                </p>
+                                <q class="title">Jenny Martian</q>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="contact-section" class="contact-section">
+            <div class="get-touch container">
+                <div>
+                    <div class="wrapper">
+                        <h2 class="section-title">Get In Touch</h2>
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    </div>
+                    <form class="myForm">
+                        <input type="text" name="customer_name" id="customer_name" required placeholder="Name">
+                        <input type="email" name="email_address" id="email_address" placeholder="Email">
+                        <input type="tel" name="phone" id="phone" placeholder="Phone">
+                        <div class="comment-box">
+                            <textarea name="comments" id="comments" maxlength="500" placeholder="Message"></textarea>
+                        </div>
+                    </form>
+                    <div class="form-button">
+                        <button class="shoot-button">SHOOT MESSAGE</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <section id="footer" class="footer">
+        <div class="footer container">
+            <h2><b>Copyright © 2021 Talos HTML Template. All rights reserved</b></h2>
```

Comment:
don't use tags for styling, if you need to make it reflect this is important, you are already doing it by using h2